package.json

```diff
@@ -57,6 +57,7 @@
     "comment-json": "4.1.0",
     "dataobject-parser": "1.2.1",
     "deepmerge": "4.2.2",
+    "fast-glob": "^3.2.7",
```

Comment:
Introduced a new dependency for performance reasons when `depth` is specified.
For more information on the other methods we tried, see the comments in the `packages/config/src/ConfigSchema.ts` file